include/alpaka/acc/Tag.hpp

```diff
@@ -47,13 +47,13 @@ namespace alpaka
         struct AccToTag;
```

Comment:
If we follow the naming of a similar trait `EventType` https://github.com/alpaka-group/alpaka/blob/9ec06cb61e7af09a5ac934b7c4d5a28333557e87/include/alpaka/event/Traits.hpp#L22
then `AccToTag` should be `TagType` and 

```
template<typename TAcc>
        using Tag = typename AccToTag<TAcc>::type;
```

should be

```
template<typename TAcc>
using Tag = typename taris::TyagType<TAcc>::type;
```

and should be located in the alpaka namespace. @SimeonEhrig sry that I have overseen it in the original PR.

Reply:
I agree.